sdk/core/Azure.Core/perf/EventSourceBenchmark.cs

```diff
@@ -56,104 +55,35 @@ public void CleanUp()
             _eventSource.Dispose();
         }
 
-        [Benchmark(Baseline = true, Description = "Old implementation")]
-        public void OldImplementation_CallFormatting()
+        // Benchmarks
+        [Benchmark]
+        public void OldImplementation()
         {
-            _eventSource.RequestOld(Sanitizer.SanitizeUrl(_uri.ToString()), _iteration++, _iteration, FormatHeaders());
+            _eventSource.RequestOld(_sanitizedUri, _iteration++, _iteration, _headersBytes);
         }
 
-        [Benchmark(Description = "New implementation, includes reformatting")]
-        public void NewImplementation_CallFormatting()
+        [Benchmark]
+        public void NewImplementation()
         {
-            _eventSource.RequestNew(Sanitizer.SanitizeUrl(_uri.ToString()), _iteration++, _iteration, FormatHeaders());
+            _eventSource.RequestNew(_sanitizedUri, _iteration++, _iteration, _headersBytes);
         }
 
-        [Benchmark(Description = "New implementation, no reformatting")]
+        [Benchmark]
```

Comment:
It seems like since the sanitizing/formatting headers was moved to set up, this "after formatting" benchmark might not be needed?